src/client/game.cpp

```diff
@@ -4150,7 +4158,7 @@ void Game::readSettings()
 
 	m_cache_fog_start = rangelim(m_cache_fog_start, 0.0f, 0.99f);
 	m_cache_cam_smoothing = rangelim(m_cache_cam_smoothing, 0.01f, 1.0f);
-	m_cache_mouse_sensitivity = rangelim(m_cache_mouse_sensitivity, 0.001, 100.0);
+	m_cache_mouse_sensitivity = rangelim(m_cache_mouse_sensitivity, 0.0f, std::numeric_limits<float>::max());
```

Comment:
This change is nonsense. Even if you *ever* (rather never) need more than factor 100, use `std::max` instead.

Reply:
sorry, i made this fork a long time ago and i wasn't really considering a pull request, it should absolutely be updated